python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -95,13 +95,21 @@ class CoaddAnalysisConfig(Config):
     fluxToPlotList = ListField(dtype=str, default=["base_GaussianFlux", "base_CircularApertureFlux_12_0",
                                                    "ext_photometryKron_KronFlux", "modelfit_CModel"],
                                doc="List of fluxes to plot: mag(flux)-mag(base_PsfFlux) vs mag(fluxColumn)")
-    columnsToCopy = ListField(dtype=str,
-                              default=["calib_psf_used", "calib_psf_candidate", "calib_psf_reserved",
-                                       "calib_astrometry_used", "calib_photometry_used",
-                                       "calib_photometry_reserved", "detect_isPatchInner",
-                                       "detect_isTractInner", "merge_peak_sky", "calib_psfUsed",
-                                       "calib_psfCandidate", ],
-                              doc="List of columns to copy from one source catalog to another.")
+    # We want the following to come from the *_meas catalogs as they reflect
+    # what happened in SFP calibration.
+    columnsToCopyFromMeas = ListField(dtype=str, default=["calib_",],
+                                      doc="List of \"startswith\" strings of column names to copy from "
+                                      "*_meas to *_forced_src catalog.  All columns that start with one "
+                                      "of these strings will be copied from the *_meas catalog into the "
+                                      "*_forced_src catalog.")
```

Comment:
This confused me initially because I wasn't familiar with startswith, might be easier if it said, "list of prefixes of column names to be used with `startswith` ..", or something similar.

Reply:
Amended doc to omit use of `startswith`.